.idea/comfort_tool.iml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't think any of these files related to your IDE should be included.. there are much lighter weight ways of running tests in python in a shell. I think you could add `.idea/` to the gitignore

Reply:
I have added the .idea folder to .gitignore, however, the folder it is still there. I will try to solved that issue.

Reply:
Added the .idea file to .gitignore and removed from the GitHub repository